src/fs/copy.js

```diff
@@ -1,5 +1,31 @@
+import { existsSync, mkdirSync, readdirSync, copyFileSync } from 'fs';
```

Comment:
I think it is better practice to use 'fs/promises' only (except of `existsSync` method)

Reply:
@lazy-goose Please, don't forget to put your score through Cross-Check: Review Form